monai/apps/pathology/transforms/post/array.py

```diff
@@ -624,3 +625,70 @@ def __call__(  # type: ignore
         type_prob = type_dict[inst_type] / (sum(seg_map_crop) + 1.0e-6)
 
         return (int(inst_type), float(type_prob))
+
+
+class HoVerNetNCBranchPostProcessing(Transform):
+    """
+    The whole post-procesing transform for nuclear type classification branch. It return a dict which contains
+    centroid, bounding box, type prediciton for each instance.
+
+    Args:
+        min_num_points: assumed that the created contour does not form a contour if it does not contain more points
+            than the specified value. Defaults to 3.
+        level: optional. Used in `skimage.measure.find_contours`. Value along which to find contours in the array.
+            By default, the level is set to (max(image) + min(image)) / 2.
+        return_centroids: whether to return centroids for each instance.
+        output_classes: number of the nuclear type classes.
+    """
+
+    def __init__(
+        self,
+        min_num_points: int = 3,
+        level: Optional[float] = None,
+        return_centroids: Optional[bool] = False,
+        output_classes: Optional[int] = None,
+    ) -> None:
+        super().__init__()
+        self.return_centroids = return_centroids
+        self.output_classes = output_classes
+
+        self.generate_instance_contour = GenerateInstanceContour(min_num_points=min_num_points, level=level)
+        self.generate_instance_centroid = GenerateInstanceCentroid()
+        self.generate_instance_type = GenerateInstanceType()
+
+    def __call__(self, type_pred: NdarrayOrTensor, inst_pred: NdarrayOrTensor) -> Dict:  # type: ignore
```

Comment:
```suggestion
    def __call__(self, type_pred: NdarrayOrTensor, instance_pred: NdarrayOrTensor) -> Dict:  # type: ignore
```

Reply:
Please try to avoid non-common abbreviations and shortened words (especially where exposed to the users). Thanks

Reply:
Thanks! Have updated in the latest commit.